third_party/build/bin/grpc_build.sh

```diff
@@ -64,13 +65,16 @@ git submodule update --init
 # change default prefix from /usr/local to /tmp/build-grpc-dev/install/usr/local
 sed -i 's/.usr.local$/\/tmp\/build-grpc-dev\/install\/usr\/local/' Makefile
 
-# build and install grpc
+mkdir -p cmake/build
```

Comment:
What is the advantage of having the cmake build in a separate directory, and not just doing it in the `${WORK_DIR}`?
This seems to add unnecessary complexity to me.